streams/src/main/java/org/apache/kafka/streams/state/internals/metrics/StateStoreMetrics.java

```diff
@@ -307,6 +315,26 @@ public static Sensor rangeSensor(final String threadId,
         );
     }
 
+    public static Sensor prefixScanSensor(final String threadId,
+                                     final String taskId,
+                                     final String storeType,
+                                     final String storeName,
+                                     final StreamsMetricsImpl streamsMetrics) {
+        return throughputAndLatencySensor(
```

Comment:
For new sensors like this, we only need to consider built-in metrics version `LATEST`. Hence, you should not call `throughputAndLatencySensor()`, but only call the parts that are relevant for `LATEST` and not for `FROM_0100_TO_24`. You also need to adapt the corresponding unit test for that. See also KIP-444 for more details.